src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,130 @@
-import { PeopleFilters } from './PeopleFilters';
-import { Loader } from './Loader';
-import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
 
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { getFilteredPeople } from '../utils/getFilteredPeople';
+import { personsWithParents } from '../utils/PeopleData';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { PeopleFilters } from './PeopleFilters';
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  const { slug } = useParams();
+
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+
+  const sex = searchParams.get('sex') || '';
+
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
these are the same type of constants, therefore those spaces don't make much sense